pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       containers:
       - name: wso2apim-with-analytics-apim-analytics
-        image: docker.wso2.com/wso2am-analytics:2.5.0
+        image: docker.wso2.com/wso2am-analytics-worker:2.6.0
         resources:
```

Comment:
IMO, we should define the resource requirements and limits for API Manager Analytics based on the recommendations [1].

WDYT?

[1]: [Resource requirements and limits](https://github.com/wso2/kubernetes-apim/blob/2.2.x/pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml#L56)